build.gradle

```diff
@@ -87,9 +82,29 @@ project('ripc-protocol-tcp') {
 	dependencies {
 		// ripc-core
 		compile project(":ripc-core")
+        compile "org.slf4j:slf4j-api:1.7.6"
+		testCompile 'io.reactivex:rxjava:1.0.8'
```

Comment:
Aah this was a left-over from me having the example (as a test) inside ripc-protocol-tcp demonstrating the usage. This is a test dep though. I will remove it, it isn't used.